features/internal/electricalconnection_test.go

```diff
@@ -549,25 +549,77 @@ func (s *ElectricalConnectionSuite) Test_GetLimitsForParameterId() {
 	assert.Equal(s.T(), defaultV, 0.1)
 }
 
+func (s *ElectricalConnectionSuite) Test_GetLimitsForParameterId_Elli_1Phase() {
+	filter := model.ElectricalConnectionPermittedValueSetDataType{
+		ParameterId: util.Ptr(model.ElectricalConnectionParameterIdType(0)),
+	}
+	minV, maxV, defaultV, err := s.localSut.GetPermittedValueDataForFilter(filter)
+	assert.NotNil(s.T(), err)
+	assert.Equal(s.T(), minV, 0.0)
+	assert.Equal(s.T(), maxV, 0.0)
```

Comment:
I think you could simplify as

    s.Equal(minV…)

Reply:
I like things to be explicit. But thanks for the hint.

Reply:
Imho that is explicit since the Suite already knows the testing.T. Fwiw, with require it‘s a bit different:

    s.Require().Equal(…)